doc/source/reference/random/index.rst

```diff
@@ -7,58 +7,146 @@
 
-Numpy's random number routines produce pseudo random numbers using
-combinations of a `BitGenerator` to create sequences and a `Generator`
-to use those sequences to sample from different statistical distributions:
-
-* BitGenerators: Objects that generate random numbers. These are typically
-  unsigned integer words filled with sequences of either 32 or 64 random bits.
-* Generators: Objects that transform sequences of random bits from a
-  BitGenerator into sequences of numbers that follow a specific probability
-  distribution (such as uniform, Normal or Binomial) within a specified
-  interval.
-
-Since Numpy version 1.17.0 the Generator can be initialized with a
-number of different BitGenerators. It exposes many different probability
-distributions. See `NEP 19 <https://www.numpy.org/neps/
-nep-0019-rng-policy.html>`_ for context on the updated random Numpy number
-routines. The legacy `RandomState` random number routines are still
-available, but limited to a single BitGenerator. See :ref:`new-or-different` 
-for a complete list of improvements and differences from the legacy
-``RandomState``.
-
-For convenience and backward compatibility, a single `RandomState`
-instance's methods are imported into the numpy.random namespace, see
-:ref:`legacy` for the complete list.
-
 .. _random-quick-start:
 
 Quick Start
 -----------
 
-Call `default_rng` to get a new instance of a `Generator`, then call its
-methods to obtain samples from different distributions.  By default,
-`Generator` uses bits provided by `PCG64` which has better statistical
-properties than the legacy `MT19937` used in `RandomState`.
+The :mod:`numpy.random` module implements pseudo-random number generators
+(PRNGs) with the ability to draw samples from a variety of probability
+distributions. In general, users will create a `Generator` instance with
+`default_rng` and call the various methods on it to obtain samples from
+different distributions.
+
+::
+
+  >>> import numpy as np
+  >>> rng = np.random.default_rng()
+  # Generate one random float uniformly distributed over the range [0, 1)
+  >>> rng.random()  #doctest: +SKIP
+  0.06369197489564249  # may vary
+  # Generate an array of 10 numbers according to a unit Gaussian distribution.
+  >>> rng.standard_normal(10)  #doctest: +SKIP
+  array([-0.31018314, -1.8922078 , -0.3628523 , -0.63526532,  0.43181166,  # may vary
+          0.51640373,  1.25693945,  0.07779185,  0.84090247, -2.13406828])
+  # Generate an array of 5 integers uniformly over the range [0, 10).
+  >>> rng.integers(low=0, high=10, size=5)  #doctest: +SKIP
+  array([8, 7, 6, 2, 0])  # may vary
+
+PRNGs are deterministic sequences and can be reproduced by specifying a seed to
+control its initial state. By default, with no seed, `default_rng` will create
+the PRNG using nondeterministic data from the operating system and therefore
+generate different numbers each time. The pseudorandom sequences will be
+practically independent.
```

Comment:
Which statement specifically? "practically independent" instead of "independent"? Pseudorandom sequences are never actually fully independent. There's always an algorithmic relationship. That's why they are "pseudo". What I meant by "practically independent" was meant to convey "independent for all purposes for which a pseudorandom sequence is acceptable in the first place".

Reply:
I think all of that is too complicated and not necessary for newcomers. I was even considering commenting about the use of the worlds "sample", "draw" and "pseudo" above. This page being the first entry point to random numbers for a lot of folks (and not just folks from the scientific community, as NumPy is widely used in a lot of domains) and as such should be really basic to me. I don't think it helps much to add such details or nuances (yes "practically") here and most a lot of that could be put further down the road.

Reply:
I am firm on introducing this as "pseudo-random number generation" in the first sentence (following the [example of the stdlib](https://docs.python.org/3/library/random.html)). People need to know that term so that they can connect our functionality to what else they read about PRNGs to understand if it fits their problem. I am okay with dropping the "pseudo" later for flow. I am not okay with just claiming "independence"; from experience, we get dinged on such loose claims whenever we state that.

Reply:
I would not say everyone "need" to know such terms to use the API. But yes, looking for some "common" terms in the doc should point to somewhere.

I am pretty sure most people have no clue about ins and outs of PRNGs and just want numbers. Which is fine, and that was my point here. There is not necessary a need to use a lot of jargon in the first sentences.

Again, just wanted to raise this and will let the doc team decide.

Reply:
Resolving, this seems fine as is.